schemas/TransferTemplate.json

```diff
@@ -65,6 +61,21 @@
       "minItems": 1,
       "uniqueItems": true,
       "items": { "$ref": "FundsTemplate.json" }
+    },
+    "additional_info": {
+      "description": "Includes part_of_payment, case, etc.",
+      "type": "object",
+      "properties": {
+        "part_of_payment": {
```

Comment:
The payment ID is going to be attached to the transfer regardless, so why not use a descriptive name? Its optional and wrapped inside `additional_info`, and the ledger code never references it anyway.